nav2_behavior_tree/CMakeLists.txt

```diff
@@ -53,6 +53,9 @@ ament_target_dependencies(${library_name}
 add_library(nav2_compute_path_to_pose_action_bt_node SHARED plugins/action/compute_path_to_pose_action.cpp)
 list(APPEND plugin_libs nav2_compute_path_to_pose_action_bt_node)
 
+add_library(nav2_update_goal_pose_action_bt_node SHARED plugins/action/update_goal_pose_action.cpp)
```

Comment:
Can we add the word "dynamically" to the naming here to make it clear its for dynamic goal updating in a single task